### PR TITLE
Simplify description for VirtualBox

### PR DESCRIPTION
**Pull request by xasx**

### CODE REVIEW

File: plugins/oraclevirtualbox.plugin/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"icon": "virtualbox",
 	"label": "VirtualBox",
-	"description": "VirtualBox is an hypervisor software to create virtual machines.",
+	"description": "VirtualBox is a hypervisor software to create virtual machines.",
```
Comment (bddenhartog) [inline plugins/oraclevirtualbox.plugin/metadata.json]: "VirtualBox **is a hypervisor software**..." reads funny to me. How about we try:

_VirtualBox is a virtual machine hypervisor_

Unless you think this is _too_ concise, and have another idea.

**Comment by xasx** [comment]
@bddenhartog I changed the description following your suggestion. Please check.

**Comment by bddenhartog** [comment]
@xasx Sorry it took a while to get to this, it's been a hectic few weeks.

Thanks for the patch, and welcome to Fedy!

**Comment by xasx** [comment]
@bddenhartog never mind. Thanks for merging 😃

**Comment by bddenhartog** [comment]
No problem. I'm currently in the middle of automating the build and deployment process, so unfortunately I won't be doing a release for a week or two.

**Pull request merged**